docs/SyncVsAsync.md

```diff
@@ -0,0 +1,67 @@
+---
+id: sync-vs-async
+title: Sync mode vs async
+---
+
+## Async mode
+
+Every browser/element function call returns a Promise and have to be awaited to get result.
```

Comment:
Maybe this instead:
Every browser/element function call returns a Promise and needs to be awaited to get result.